pkg/git/repo.go

```diff
@@ -77,7 +77,7 @@ func (r *repo) GetClonedBranch() string {
 
 // Copy does copying the repository to the given destination.
 func (r *repo) Copy(dest string) (Repo, error) {
```

Comment:
@nghialv btw, perhaps was this method originally created for copying the repo root to the given `dest` as a subdirectory?

Reply:
@nghialv If so, my usage is just evil.

Reply:
But even if so, it's not appropriate to give back Repo. Well, it's nothing!

Reply:
The `Copy` function copies all the data of the original repository into a new directory.
And it expects the destination directory was not created before passing to this function.
I am sorry for the lack of explanation. We should have at least a comment.


From the current situation, I think we have 2 options:
- keep the current behavior and add a comment to explain that the dest directory must not be created before
- require the user to ensure that the directory must be created before passing to this function and change the command to `cp -rf src/* dest`

Reply:
One vote for the first option. The behaviors are slightly different depending on the `cp` command implementation (mostly BSD and GNU). Actually `cp -rf src/* dest` doesn't copy the hidden files including `.git/` and so on.

Let me keep the current behavior and just add a comment!

Reply:
👍